### PR TITLE
feat: add VTR watcher

### PR DESCRIPTION
**Pull request by fjsandov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,11 +32,21 @@ async function watchGithubNotifications() {
...
+}
+
 [
   /* 
     Add other watchWatcher here to develop
     Comment to avoid calling
   */
   // watchExample,
-  // watchGithubNotifications
+  // watchGithubNotifications,
```
Comment (negebauer) [inline index.js]:
```suggestion
  // watchGithubNotifications
```

File: vtr/vtr-plans.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  const price = extractPlanPrice($, productElem);
+  const details = extractPlanDetails($, productElem);
+  return { name, price, details };
+}
+
+const url =
+  "https://vtr.com/productos/HogarPacks/triple-pack-banda-ancha-television-telefonia";
+
+async function fetchPlans(axios, cheerio) {
+  const response = await axios.get(url);
```
Comment (negebauer) [inline vtr/vtr-plans.js]: `try catch`

File: vtr/vtr-plans.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    priceString = $planElement
+      .find(".box-plp-price-ab > .price-plp > h5 > strong")
+      .text()
+      .trim();
+  }
+  return Number(priceString.replace("$", "").replace(".", ""));
+}
+
+function extractPlanName($, planElem) {
+  // const productName = $(planElem).find('.terms-icon-list > .new-product-wrap > .product-name').text().trim();
```
Comment (negebauer) [inline vtr/vtr-plans.js]: why this line? add another comment explaining why?

File: vtr/vtr-plans.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  const omProductName = $(planElem)
+    .find("a.om-C2C")[0]
+    .attribs["om-productname"].trim();
+  // console.log('ProductName:', productName, '---', omProductName);
```
Comment (sasalatart) [inline vtr/vtr-plans.js]: For debug purposes?

File: vtr/vtr-plans.js [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+function extractPlan($, productElem) {
+  const name = extractPlanName($, productElem);
+  const price = extractPlanPrice($, productElem);
+  const details = extractPlanDetails($, productElem);
+  return { name, price, details };
+}
+
+const url =
```
Comment (sasalatart) [inline vtr/vtr-plans.js]: Probably best to use `const URL` for these kind of consts, or not?

File: vtr/index.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    (planA, planB) => planEquals(_, planA, planB)
+  );
+  const notifications = newPlans.map(newPlan => ({
+    key: notificationTypes.newPlan.key,
+    message: `VTR has a new plan: ${newPlan.name}`,
+  }));
+  return { snapshot, notifications };
+}
+
+module.exports = watch;
```
Comment (negebauer) [inline vtr/index.js]:
```js
module.exports = {
  config,
  watch,
}```

File: index.js [hunk 7]
```diff
@@ -35,6 +36,15 @@ async function watchGithubNotifications() {
   console.log("notifications:", notifications);
 }
 
+// eslint-disable-next-line no-unused-vars
+async function watchVtrPlans() {
+  const { snapshot, notifications } = await execute(vtrPlansWatcher, {
```
Comment (negebauer) [inline index.js]:
```suggestion
  const { snapshot, notifications } = await execute(vtrPlansWatcher.watch, {
```

**Pull request merged**